cmd/extracting/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM golang:1.9.2 as builder
+WORKDIR /home/src
+COPY . /home/src
+RUN go get github.com/dustin/go-humanize
```

Comment:
What is this being used for? If this is OSS needed for the demo it needs to be registered.

Reply:
This is removed now